### PR TITLE
chore: update to angular 2.0.0-beta.0

### PR DESCRIPTION
**Pull request by Foxandxss**
Decided to pull out all the other dependencies too for the npm3 users.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,9 +37,12 @@
...
     "karma-sourcemap-loader": "^0.3.6",
     "merge2": "^0.3.6",
+    "reflect-metadata": "^0.1.2",
     "run-sequence": "^1.1.4",
+    "rxjs": "^5.0.0-beta.0",
     "systemjs": "^0.19.5",
     "typescript": "^1.6.2",
-    "webpack": "^1.12.2"
+    "webpack": "^1.12.2",
+    "zone.js": "^0.5.10"
```
Comment (pkozlowski-opensource) [inline package.json]: Lock the zone, don't take chances :-)

**Comment by pkozlowski-opensource** [comment]
LGTM, generally speaking, but please follow the same patterns for deps as in ng2. We don't want to take chances, especially with zone.

**Comment by Foxandxss** [comment]
Locked all the new stuff. I don't want to mess with rxjs versions either.

**Comment by Foxandxss** [comment]
Read you wrongly. Now it is ng2 pattern.

**Comment by pkozlowski-opensource** [comment]
LGTM if TravisCI is happy

**Pull request closed without merging**